docker/ftest-boot2docker/src/test/java/org/arquillian/cube/persistence/UserRepositoryTest.java

```diff
@@ -1,17 +1,20 @@
 package org.arquillian.cube.persistence;
 
-import java.io.IOException;
 import javax.inject.Inject;
+import java.io.IOException;
+import org.arquillian.cube.remote.requirement.RequiresRemoteResource;
 import org.jboss.arquillian.container.test.api.Deployment;
 import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
 import org.jboss.shrinkwrap.api.asset.StringAsset;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
 import org.junit.runner.RunWith;
 
 @RunWith(Arquillian.class)
+@Category({RequiresRemoteResource.class})
```

Comment:
This allows tests to be excluded that make real remote calls, which just breaks when behind a corporate firewall.